code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -349,6 +349,7 @@
 	equip_delay_self = 50
 	strip_delay = 60
 	breakouttime = 3000
+	pocket_storage_component_path = FALSE

```

Comment:
Are you sure this is supposed to be `FALSE` instead of `null`?

Reply:
I mean, null would likely work as well, but the code seems to work as is

Reply:
Did you check the View Runtimes verb for any runtimes when you were testing a subtype set to FALSE? If there wasn't any runtimes I'll sign off on this PR.